File: app/src/main/java/com/piyush/barclays/response/chart/Quote.java

package com.piyush.barclays.response.chart;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Quote {

    @SerializedName("volume")
    @Expose
    private List<Integer> volume = null;
    @SerializedName("open")
    @Expose
    private List<Double> open = null;
    @SerializedName("low")
    @Expose
    private List<Double> low = null;
    @SerializedName("close")
    @Expose
    private List<Double> close = null;
    @SerializedName("high")
    @Expose
    private List<Double> high = null;

    public List<Integer> getVolume() {
        return volume;
    }

    public void setVolume(List<Integer> volume) {
        this.volume = volume;
    }

    public List<Double> getOpen() {
        return open;
    }

    public void setOpen(List<Double> open) {
        this.open = open;
    }

    public List<Double> getLow() {
        return low;
    }

    public void setLow(List<Double> low) {
        this.low = low;
    }

    public List<Double> getClose() {
        return close;
    }

    public void setClose(List<Double> close) {
        this.close = close;
    }

    public List<Double> getHigh() {
        return high;
    }

    public void setHigh(List<Double> high) {
        this.high = high;
    }

}

File: app/src/main/java/com/piyush/barclays/response/search/Quote.kt
package com.piyush.barclays.response.search

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


class Quote {
    @SerializedName("exchange")
    @Expose
    var exchange: String? = null

    @SerializedName("shortname")
    @Expose
    var shortname: String? = null

    @SerializedName("quoteType")
    @Expose
    var quoteType: String? = null

    @SerializedName("symbol")
    @Expose
    var symbol: String? = null

    @SerializedName("index")
    @Expose
    var index: String? = null

    @SerializedName("score")
    @Expose
    var score: Double? = null

    @SerializedName("typeDisp")
    @Expose
    var typeDisp: String? = null

    @SerializedName("longname")
    @Expose
    var longname: String? = null

    @SerializedName("isYahooFinance")
    @Expose
    var isYahooFinance: Boolean? = null
}
